Doc/using/configure.rst

```diff
@@ -94,6 +94,154 @@ List all :file:`configure` script options using::
 
 See also the :file:`Misc/SpecialBuilds.txt` in the Python source distribution.
 
+Variables
+---------
+
+.. cmdoption:: BOLT_APPLY_FLAGS
+
+   Arguments to ``llvm-bolt`` when creating a `BOLT optimized binary
+   <https://github.com/facebookarchive/BOLT>`_.
+
+.. cmdoption:: BOLT_INSTRUMENT_FLAGS
+
+   Arguments to ``llvm-bolt`` when instrumenting binaries.
+
+.. cmdoption:: BZIP2_CFLAGS
+.. cmdoption:: BZIP2_LIBS
+
+   C compiler and linker flags to link Python to ``libbz2``, used by :mod:`bz2`
+   module, overriding ``pkg-config``.
+
+.. cmdoption:: CC
+
+   C compiler command.
+
+.. cmdoption:: CFLAGS
+
+   C compiler flags.
+
+.. cmdoption:: CPP
+
+   C preprocessor command.
+
+.. cmdoption:: CPPFLAGS
+
+   C preprocessor flags, e.g. ``-I<include dir>``.
```

Comment:
Maybe we should put BOLT, CC, CFLAGS, CPP and CPPFLAGS in a "Compiler flags" separated section?